src/algebra/module/localized_module.lean

```diff
@@ -282,6 +281,156 @@ instance is_module : module (localization S) (localized_module S M) :=
   add_smul := add_smul',
   zero_smul := zero_smul' }
 
+instance is_module' : module R (localized_module S M) :=
```

Comment:
The fact that this proof is so long makes me thinking we are missing the API, especially because of all the `change`. Can you extract the various results as separate lemmas? And also add a lemma that can be used instead of `change`.

Reply:
I have removed all `change`